RecoJets/JetProducers/python/ak4PFJets_cfi.py

```diff
@@ -36,5 +36,7 @@
     jetPtMin = 100.0
 )
 from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
-pp_on_AA_2018.toModify(ak4PFJets,src = "pfNoPileUpJMEHI", inputEtMin = 9999)
-pp_on_AA_2018.toModify(ak4PFJetsCHS,src = "pfNoPileUpJMEHI", inputEtMin = 9999)
+from Configuration.Eras.Modifier_pp_on_PbPb_run3_cff import pp_on_PbPb_run3
+(pp_on_AA_2018 | pp_on_PbPb_run3).toModify(ak4PFJets, src = "pfEmptyCollection", inputEtMin = 9999)
+(pp_on_AA_2018 | pp_on_PbPb_run3).toModify(ak4PFJetsCHS, src = "pfEmptyCollection", inputEtMin = 9999)
+(pp_on_AA_2018 | pp_on_PbPb_run3).toModify(ak4PFJetsPuppi, src = "pfEmptyCollection", jetPtMin = 9999.) 
```

Comment:
shouldn't this still use `inputEtMin`?
Actually, are the modifications to the cuts really needed if the input is really an empty collection?
Better keep it minimal.